docs/en/reference/configuration.rst

```diff
@@ -324,6 +325,7 @@ pdo_oci / oci8
 -  ``driverOptions`` (array):
     -  ``exclusive`` (boolean): Once specified for an ``oci8`` connection, forces the driver to always establish
        a new connection instead of reusing an existing one from the connection pool.
+    -  ``protocol`` (string): The protocol used to connect to Oracle (default TCP, Autonomous tends to be TCPS)
```

Comment:
It looks like Autonomous is an Oracle product. Is it relevant in this documentation? An explanation of what TCPS is or a link to documentation would make more sense.

Reply:
I have updated it with a link to TCPS on Oracle.